src/content/get-started/advanced-commands-and-concepts.mdx

```diff
@@ -0,0 +1,91 @@
+
+[Add states of dev environments and how to see their logs. The same applies to each service: how to see the logs, the meaning of the states, how to get the endpoints, …]
+
+### [Development Containers](development/containers/index.mdx)
+
+Development containers are essential for creating isolated and reproducible development environments. Definitions for the Containers look like docker-compose, but run in Kubernetes. Okteto supports two modes for development containers:
+
+- **File Sync Development Mode**: Syncs your local changes to your remote development container (default mode).
+- **Hybrid Development Mode**: Runs your service locally and simulates your laptop belonging to the cluster network by using SSH tunnels. This may be helpful if you work with large file sizes in part of your application. Okteto Code Sync does not run while using hybrid mode since the changes are built locally.
+
+### [Development Images](development/images.mdx)
+
+Development images define the runtime your application runs in during development, ensuring consistency and reproducibility across different stages of your workflow, developers, and different local machines.
+
+### [Variables](core/okteto-variables.mdx#user-variables)
+
+Environment variables are useful for storing user-specific configuration values that are not shared with the rest of the team. They can also be useful when you need to specify ad-hoc variables at [the time of deployment](core/okteto-variables.mdx#deployment-variables).
+
+### Okteto Preview Environments
+
+Okteto's [Preview Environments](previews/index.mdx) automatically generate a unique, shareable version of your application for each pull request, making code reviews, automated end-to-end testing, and stakeholder feedback a breeze. [add language clarifying that platform team needs to set this up]
+
+## FAQ
+
+**When does `okteto deploy` look at the Git repo vs my own code?**
+
+- `okteto deploy` uses the code from your Git repository to deploy the application. After running `okteto up`, Okteto’s file sync will update your changes remotely
+
+**What’s the difference between `okteto up` and `okteto up --deploy`?**
+
+- `okteto up` creates the Development Environment of a service based on its previously deployed version.
+- `okteto up --deploy` runs the `okteto deploy` section first, picking up any new changes from the repository, and then connects to the Development Environment.
+
+**What does `okteto deploy --build` do?**
+
+- This command forces a rebuild of the images, incorporating any local changes. It rebuilds all images, not just the ones with changes.
+
+**What does the `--remote` flag do?**
+
+- Specifies that certain commands should be executed in the remote Development Environment instead of locally.
```

Comment:
This is not accurate. Remote execution is explained here:
https://www.okteto.com/docs/core/remote-execution/